backend/main.go

```diff
@@ -199,6 +199,7 @@ func initialize(servicePort, databaseURL, redisURL, loggingURL, statsdURL, certP
 	r.Group(func(r chi.Router) {
 		r.Get("/logout", func(w http.ResponseWriter, r *http.Request) {
 			for _, cookie := range r.Cookies() {
+				svc.Log.Info("Removing cookie: "+cookie.Name, nil)
```

Comment:
I'm interested in learning more about why this log is useful for us.

Reply:
Logout doesn't remove cookies and seems to only remove the refresh_token so I'm curious if it even sees the access_token and that's the only reason. Purely temporary for my side

Reply:
maybe `debug` instead? This would be a nice to have when doing things locally I'm sure but having this on prod grafana doesn't seem interesting.